### PR TITLE
fix: justify-content property

### PR DESCRIPTION
**Pull request by ahorek**
### Description
fix a css property according to specs, no functional changes

### Motivation and Context
+ ```justify-content: none;``` is an invalid css, see https://developer.mozilla.org/en-US/docs/Web/CSS/justify-content

### How Has This Been Tested?
tested with and without servers (firefox and chrome)

### Screenshots
obrazek

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [x] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by glowcloud** [comment]
Thanks for the suggested changes!

Could I ask you to confirm if this issue still exists in the current versions of the browsers? I could not replicate the button moving to the left when loading a specification without schemes.

File: src/style/_layout.scss [hunk 1]
```diff
@@ -844,9 +844,6 @@
         */
         .auth-wrapper 
         {
-            flex: none;
```
Comment (glowcloud) [inline src/style/_layout.scss:847]: Removing this will move the authorize button to the right on smaller screens when we have schemes. 

test_after

Can you revert this change?

**Review by glowcloud — CHANGES_REQUESTED** [review]
Thanks for the confirmation!
Please take a look at my comment.

File: src/style/_layout.scss (hunk 1 above)
Comment (ahorek) [reply src/style/_layout.scss:846]: done! I also updated the description, because the original problem no longer applies here.

**Review by char0n — APPROVED** [review]

**Review by glowcloud — APPROVED** [review]
Great, LGTM!
